sympy/core/tests/test_numbers.py

```diff
@@ -320,6 +320,16 @@ def test_Rational_new():
     assert Rational('.76').limit_denominator(4) == n3_4
     assert Rational(19, 25).limit_denominator(4) == n3_4
     assert Rational('19/25').limit_denominator(4) == n3_4
+    n2_3 = Rational(2, 3)
+    assert Rational('2/3') == n2_3
```

Comment:
Doesn't this (and the next few test) just test the parsing? Not needed to test again.